File: lesson_7/Hometask_7_1/Drink.java
package lesson_7.Hometask_7_1;

import java.util.ArrayList;
import java.util.List;

public class Drink {
    protected String name;
    protected int volume;

    List<Drink> drink = new ArrayList<>();

    Drink(String name, int volume) {
        this.name = name;
        this.volume = volume;
        drink.add(this);
    }


    public String getName() {
        return name;
    }

    public int getVolume() {
        return volume;
    }

    public String toString() {
        return String.format("%s, volume %d", name, volume);

    }

}

File: lesson_13/hometask/HW_py/Drink.py

class Drink:

    def __init__(self, name, volume):
        self._name = name
        self._volume = volume

    def getName(self):
        return self._name
    
    def getVolume(self):
        return self._volume
    
    def __str__(self):
        return f"{self._name} volume {self._volume}"
    

    